build/release.js

```diff
@@ -4,30 +4,35 @@ var fs = require( "fs" );
 
 module.exports = function( Release ) {
 
-	var
-		distFiles = [
-			"dist/jquery.js",
-			"dist/jquery.min.js",
-			"dist/jquery.min.map",
-			"dist/jquery.slim.js",
-			"dist/jquery.slim.min.js",
-			"dist/jquery.slim.min.map"
-		],
-		filesToCommit = [
-			...distFiles,
-			"src/core.js"
-		],
-		cdn = require( "./release/cdn" ),
-		dist = require( "./release/dist" ),
+	const distFiles = [
+		"dist/jquery.js",
+		"dist/jquery.min.js",
+		"dist/jquery.min.map",
+		"dist/jquery.slim.js",
+		"dist/jquery.slim.min.js",
+		"dist/jquery.slim.min.map"
+	];
+	const filesToCommit = [
+		...distFiles,
+		"src/core.js"
+	];
+	const cdn = require( "./release/cdn" );
+	const dist = require( "./release/dist" );
 
-		npmTags = Release.npmTags;
+	const npmTags = Release.npmTags;
```

Comment:
@timmywil We had a talk a while ago about the `onevar` rule in ESLint. You said you don't want to remove/change the rule so that code using the preset doesn't need to adapt. However:

1. The rule is *not* actually enabled in the jQuery preset. AFAIK it was removed because Jörn didn't want it in jQuery UI or QUnit due to Git diff issues etc.
2. While we adhere to this way of declaring variables in jQuery code, I'd argue it's actually harmful when used for `const`/`let`. This is because for `var`s we were emulating how the JS engine interprets those declarations; in case of `const`/`let` the declarations are not hoisted and it's common to put them in the middle of the function where they are needed.
3. Other ES6 code in our repo already uses separate declarations per variable so we have some history here. That also means we don't have history of requiring this rule for `const`/`let` so not applying it shouldn't cause issues with code refactorings.

Reply:
The `onevar` rule can technically differentiate between `var`, `const`, and `let`. I'd be immediately in favor of specifying that we only require `onevar` for `var`. As for `var`, I'm not sure it's worth the large diffs and blame overrides.

Reply:
@timmywil Makes sense, I didn't know it can be configured just for `var`. We can add it to our own ESLint config since it was rejected as a general rule for the jQuery preset.

Reply:
@timmywil Done in #4615.